.github/workflows/deploy_testnets.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           google-application-credentials: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
           sh-attesters: hydra-s1-accountbound
           sh-attesters-networks: goerli mumbai
-          sh-registry-tree-networks: goerli mumbai
+          # sh-registry-tree-networks: goerli mumbai
```

Comment:
Since we are running a check on the `sh-registry-tree-networks` variable in the `update-groups-send-on-chain` github action. 

```bash
- name: make groups available to hydra-s2 registry tree on chain
      shell: bash
      if: ${{ inputs.sh-registry-tree-networks }}
      ...
      run: |
        yarn make-groups-available hydra-s2 ${{ inputs.sh-registry-tree-networks }} --storage-type aws --send-on-chain --logger-type local-file ${{ inputs.sh-additional-params }}
```

I dont see how it can break the CI. Moreover, the variable wasn't even set in the `[.github/workflows/update_groups_weekly_testnets.yml](https://github.com/sismo-core/sismo-hub/pull/1883/files#diff-5dca37cb43f910f600c1ab70ea872863cd0d3a1f3fc1abc4aa9b89f0475584e2)`, so this behavior was already experimented as far as I understand.